oc-chef-pedant/spec/api/account/account_acl_spec.rb

```diff
@@ -14,7 +14,7 @@
 
     context "GET /users/<user>/_acl"  do
 
-      let(:actors) { ["pivotal", username] }
+      let(:actors) { ["pivotal", username].uniq }
```

Comment:
@stevendanna not if username is `pivotal` :) Which is the issue we were encountering, over and over.